plugin-valtimo/src/test/kotlin/com/ritense/valtimo/TestPlugin.kt

```diff
@@ -33,7 +33,11 @@ class TestPlugin {
         description = "This is a test action",
         activityTypes = [SERVICE_TASK_START]
     )
-    fun testAction(@PluginActionProperty testActionProperty: String) {
-        //do nothing
+    fun testAction(@PluginActionProperty attachmentIds: List<String>?) {
+        Companion.attachmentIds = attachmentIds
+    }
+
+    companion object {
```

Comment:
The `companion object` behaves a bit like a static. So this test might behave differently when multiple tests use this.

Better to make it an instance var, so that it resets when a new instance is created.

Reply:
Making an instance var is not possible in this case. Because every time this TestPlugin is retrieved, you get a new instance

Reply:
You could replace it with some callback functionality which you can inject from your test. But we'll fix that when we get there.